include/robots/tank/atv.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#ifndef _WIN32
+
+// BoB robotics includes
+#include "common/robo_claw.h"
+#include "robots/tank/tank_base.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+namespace BoBRobotics {
+namespace Robots {
+namespace Tank {
+//----------------------------------------------------------------------------
+// BoBRobotics::Robots::Tank::ATV
+//----------------------------------------------------------------------------
+//! An interface for large wheeled, all-terrain robot with two RoboClaw motor controllers
+class ATV
+  : public TankBase<ATV>
+{
+public:
+    ATV(const char *frontPath = "/dev/ttyACM0", const char *rearPath = "/dev/ttyACM1",
```

Comment:
You could use persistent device names for these values, in case they get enumerated in a different order (or there's another serial device plugged in). You should be able to find them under ``/dev/serial/``.

Reply:
ooh, I never knew about that mechanism. Once I get both motor controllers plugged into something I'll investigate - **hopefully** they have different serial numbers or some way to distinguish them.